src/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,15 @@
+.card,
+[class*="card--"] {
+  @include card;
+}
+
+.card__details {
```

Comment:
Compare the list that has the MP count below currently using list--details with the card implementation on a screen size of 329px. All flex items should be removed here.